src/gmp.c

```diff
@@ -9797,35 +9797,46 @@ strdiff (const gchar *one, const gchar *two)
   gchar *standard_err = NULL;
   char dir[] = "/tmp/gvmd-strdiff-XXXXXX";
   GError *error = NULL;
+  gchar *c_one, *c_two;
 
   if (mkdtemp (dir) == NULL)
     return NULL;
 
   one_file = g_build_filename (dir, "Report 1", NULL);
 
-  g_file_set_contents (one_file, one, strlen (one), &error);
+  c_one = g_strdup_printf ("%s\n", one);
+
+  g_file_set_contents (one_file, c_one, strlen (c_one), &error);
```

Comment:
Just a note: c_one could be freed here to save the calls below.

Reply:
Oh yes of course